linux-tools/prelink/prelink.sh

```diff
@@ -89,14 +89,22 @@ function tc_local_setup()
 {
 	tc_exec_or_break $REQUIRED || return
 
-	tc_exist_or_break /etc/prelink.conf /etc/sysconfig/prelink || return
+	#tc_exist_or_break /etc/prelink.conf /etc/sysconfig/prelink || return
+	tc_exist_or_break /etc/prelink.conf /usr/sbin/prelink || return
 
```

Comment:
Please Correct Summary !

In tc_exec_or_break $REQUIRED prelink check is done. 
And how about distros which has /etc/sysconfig/prelink ?  This change has to be made generic to work in all the cases.